WAL/WAL/Network/API/History/HistoryAPI.swift

```diff
@@ -59,15 +60,15 @@ final class HistoryAPI {
         }
     }
     
-    public func deleteHistoryData(postId: Int, completion: @escaping ((GenericResponse<DeleteHistoryResponse>?, Int?) -> ())) {
-        historyProvider.request(.deleteReserve(postId: postId)) { result in
+    public func deleteHistoryData(reservationId: Int, completion: @escaping ((DefaultResponse?, Int?) -> ())) {
+        historyProvider.request(.deleteReserve(reservationId: reservationId)) { result in
             switch result {
             case .success(let response):
+                print("✅")
                 do {
-                    self.deleteHistoryData = try response.map(GenericResponse<DeleteHistoryResponse>?.self)
+                    self.deleteHistoryData = try response.map(DefaultResponse?.self)
                     guard let deleteHistoryData = self.deleteHistoryData else { return }
                     completion(deleteHistoryData, nil)
-                    
                 } catch(let err) {
                     print(err.localizedDescription, 500)
```

Comment:
```suggestion
                    print(err.localizedDescription, err.response.statusCode)
```
이런 식으로 500이 아닐 수도 있어서 에러에서 상태 값 받아서 print 찍는게 더 명시적일 것 같아여

Reply:
미안합니다 이거 에러가 납니다 b/c err does not have response